native/src/modules/layout/components/Header.js

```diff
@@ -70,31 +67,32 @@ type PropsType = {|
   goToLanguageChange?: () => void,
   routeCityModel?: CityModel,
   language: string,
+  shareUrl: string,
   dispatch: Dispatch<StoreActionType>
 |}
 
-class Header extends React.PureComponent<PropsType> {
-  canGoBackInStack (): boolean {
-    return !!this.props.previous
-  }
+const Header = (props: PropsType) => {
+  const canGoBackInStack = useCallback((): boolean => {
+    return !!props.previous
+  })
```

Comment:
I think the `props` or `previous` dependency missing here

Reply:
Good catch. Maybe we can add this to linting?

Reply:
There is already a linting rule but just a warning yet. I think there are cases where not including the dependencies is intentional so I am not sure if we want to change it.